lib/block.ml

```diff
@@ -16,6 +16,21 @@
 
 open Util.Result.Infix
 
+let loc_error ~loc fmt =
+  Format.kasprintf
+    (fun s -> Error (`Msg s))
+    ("%a: invalid code block: " ^^ fmt)
+    Stable_printer.Location.print_loc loc
```

Comment:
I wonder, would it not be reasonable to rename `print_loc` into `pp`? That way it is very clear what it is.

Reply:
Yeah it would make more sense indeed!